docker/bash.sh

```diff
@@ -46,6 +46,16 @@ if [[ "$1" == "--net=host" ]]; then
     shift 1
 fi
 
+# Mount external directory to the docker
+CI_DOCKER_MOUNT_DIR=( )
+CI_DOCKER_MOUNT_CMD=""
+if [[ "$1" == "--mount" ]]; then
+    shift 1
+    CI_DOCKER_MOUNT_DIR+="$1"
+    shift 1
+    CI_DOCKER_MOUNT_CMD="--mount type=bind,source=${CI_DOCKER_MOUNT_DIR},target=${CI_DOCKER_MOUNT_DIR}"
```

Comment:
changed the `CI_DOCKER_MOUNT_CMD `.
Regarding to -v vs mound, docker website recommends using mount option since it has a better readable syntax and easier to use. But I don't have a preference.

Reply:
i think we should maintain one convention for this file. since we are bind-mounting, it's probably fine to just keep using `-v`. but if you want to change the others for readability, you can too.

Reply:
makes sense. Changed it to -v.
thanks!